ref/general/mp-openapi.adoc

```diff
@@ -75,27 +75,36 @@ For example, the `@Operation` annotation provides more meaningful information ab
               $ref: '#/components/schemas/InventoryList'
 ----
 
-The information provided through the OpenAPI annotations augment the basic API documentation that is generated by the JAX-RS framework.
+The information that is provided through the OpenAPI annotations augments the basic API documentation that is generated by the JAX-RS framework.
 
-See the link:https://www.openliberty.io/docs/ref/microprofile/3.0/#package=org/eclipse/microprofile/openapi/annotations/package-frame.html&class=org/eclipse/microprofile/openapi/annotations/Operation.html[MicroProfile OpenAPI Javadoc] for the annotations available.
+See here for the link:https://github.com/eclipse/microprofile-open-api/wiki/Annotation-Samples[annotations available].
 
 You can further use link:http://download.eclipse.org/microprofile/microprofile-open-api-1.1/microprofile-openapi-spec.html#_filter[filters] to organize your APIs, for example, choose to expose some APIs to one audience and other APIs to another audience.
```

Comment:
I think you should add a period after "APIs" here,  and start a new sentence with "for example" like, "For example, you can choose to expose some APIs to one audience and other APIs to another audience. The current sentence reads a bit awkwardly.